docker/build-docker-file-src.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+cat centos7 centos7-src > build-centos7-src
```

Comment:
Why do we need to concatenate these 2 files ?

Why not add file `build-centos7-src` directly to the repo ?

Reply:
`centos7-src` file contains only `RUN` docker command needed to get project sources.
It's used only in case of release to have a docker image with already cloned repository. 
`cat` is used to avoid repeating contents of `centos7` docker file in `centos7-src`. We use the concatenation of both file when building the docker image.